client/pages/details.jsx

```diff
@@ -147,26 +178,47 @@ export default class Details extends React.Component {
         </div>
         <div className='color-text-white col-md-6 font-lig'>
           <div className='display-flex space-between align-center'>
-            <h1 className='color-text-lightblue margin-top-small font-lig font-size-large'>{`COMMENTS(${this.state.comments.length})`}</h1>
+            <h1 className='color-text-lightblue margin-top-small font-lig font-size-large'>{`REVIEWS(${this.state.comments.length})`}</h1>
             <i onClick={this.handleClick} className="fa-solid fa-plus"></i>
           </div>
           <div>
             {
             this.state.comments.length > 0 && (
               this.state.comments.map((comment, index) => {
+                const date = new Date(comment.createdAt);
+                const formattedDate = `${(date.getMonth() + 1)}/${date.getDate()}/${date.getFullYear()}`;
                 return (
                 <div key = { index }>
-                <h1 className='color-text-lightblue font-small font-roboto margin-bot-user'>Alveezy</h1>
-                <hr className='spacer-line'/>
-                <p className='font-very-small font-inter margin-top-user'>  {comment.content}</p>
+                    <h1 className='color-text-lightblue fs-6 font-roboto margin-bot-user display-flex align-center'>{`Alveezy - ${formattedDate} - `}
+                  {
+                    new Array(5).fill().map((star, index) => {
+                      index += 1;
+                      return (
+                        <i key={index} className={index <= comment.ratingValue ? 'fa-solid fa-star gold star-size-small' : 'fa-regular fa-star star-size-small'}></i>
+                      );
+                    })
+                  }
+                  </h1>
+                  <hr className='spacer-line'/>
+                  <p className='font-very-small font-inter margin-top-user'>  {comment.content}</p>
                 </div>
                 );
               }))
             }
           </div>
           <Form onSubmit={this.handleSubmit} className={revealedForm}>
             <Form.Group className="mb-3" controlId="exampleForm.ControlTextarea1">
-              <Form.Control onChange={this.handleChange} as="textarea" rows={3} />
+              <div className='mb-1'>
+                {
+                  new Array(5).fill().map((star, index) => {
+                    index += 1;
+                    return (
+                        <i key={index} data-index={index} onClick={this.handleClickStar} className={index <= this.state.rating ? 'fa-solid fa-star gold' : 'fa-regular fa-star'}></i>
+                    );
+                  })
+                }
```

Comment:
Also, try making a StarRating component that you can pass onClick, and rating to so that you can eliminate some duplication in your code.

```jsx
<StarRating rating={averageRating} onClick={this.handleStarClick} />
```